torch/csrc/Module.cpp

```diff
@@ -604,6 +604,61 @@ PyObject* THPModule_userEnabledMathSDP(PyObject* _unused, PyObject* noargs) {
   else
     Py_RETURN_FALSE;
 }
+PyObject* THPModule_setGlobalCtx(PyObject* _unused, PyObject* arg) {
```

Comment:
I think some tests exercising the error cases could also be good

Reply:
Added test_opctl and some fastpath runs of actual models.